pkgs/build-support/cc-wrapper/default.nix

```diff
@@ -13,6 +13,7 @@
 , extraPackages ? [], extraBuildCommands ? ""
 , isGNU ? false, isClang ? cc.isClang or false, gnugrep ? null
 , buildPackages ? {}
+, passthru ? {}
```

Comment:
This actually isn't needed anymore, although it might still be nice to get androidndk out of stdenv.